apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -396,8 +396,8 @@ suite('ActivityHandler', function() {
 
       test('phone in notification title when contact without name', function() {
         sinon.assert.called(sendSpy);
-        var notification = sendSpy.firstCall.thisValue;
-        assert.equal(notification.title, phoneNumber);
+        var notification = sendSpy.firstCall;
+        assert.equal(notification.args[0], phoneNumber);
```

Comment:
nit: use `sinon.assert.calledWith(sendSpy, phoneNumber)`. You can even remove the first line as well.